src/prommis/nanofiltration/diafiltration_two_salt.py

```diff
@@ -0,0 +1,1097 @@
+Parameter          Value     Units
+================== ========= ================
+:math:`\beta_0`    -3.87e-06 :math:`m^2/h`
+:math:`\beta_1`    -6.56e-08 :math:`m^5/kg/h`
+:math:`\beta_2`    2.58e-08  :math:`m^5/kg/h`
+:math:`\beta_3`    -4.50e-07 :math:`m^2/h`
+:math:`\beta_4`    -1.70e-07 :math:`m^5/kg/h`
+:math:`\beta_5`    6.67e-08  :math:`m^5/kg/h`
+:math:`\beta_6`    -6.47e-07 :math:`m^2/h`
+:math:`\beta_7`    4.10e-08  :math:`m^5/kg/h`
+:math:`\beta_8`    -1.61e-08 :math:`m^5/kg/h`
+:math:`\beta_9`    -3.56e-06 :math:`m^2/h`
+:math:`\beta_{10}` 3.91e-07  :math:`m^5/kg/h`
+:math:`\beta_{11}` -1.53e-07 :math:`m^5/kg/h`
+================== ========= ================
+
+No applied potential on the system:
+
+.. math:: j_{\mathrm{Cl^-}}(\bar{x}) = - \frac{z_{\mathrm{Li^+}}}{z_{\mathrm{Cl^-}}} \frac{MW_{\mathrm{Cl^-}}}{MW_{\mathrm{Li^+}}} j_{\mathrm{Li^+}}(\bar{x}) - \frac{z_{\mathrm{Co^{2+}}}}{z_{\mathrm{Cl^-}}} \frac{MW_{\mathrm{Cl^-}}}{MW_{\mathrm{Co^{2+}}}} j_{\mathrm{Co^{2+}}}(\bar{x})
+
+Electroneutrality in the membrane:
+
+.. math:: c_{\mathrm{Cl^-},m}(\bar{x},\bar{z}) = -\frac{z_{\mathrm{Li^+}}}{z_{\mathrm{Cl^-}}} \frac{MW_{\mathrm{Cl^-}}}{MW_{\mathrm{Li^+}}} c_{\mathrm{Li^+},m}(\bar{x},\bar{z}) -\frac{z_{\mathrm{Co^{2+}}}}{z_{\mathrm{Cl^-}}} \frac{MW_{\mathrm{Cl^-}}}{MW_{\mathrm{Co^{2+}}}} c_{\mathrm{Co^{2+}},m}(\bar{x},\bar{z}) \forall \bar{z} \neq 0
+
+Mass balance (via convection) on the permeate outlet:
+
+.. math:: q_p(\bar{x}) = \bar{x} wL J_w(\bar{x}) \forall \bar{x} \neq 0
+.. math:: c_{\mathrm{Li^+},p}(\bar{x}) = \frac{j_{\mathrm{Li^+}}(\bar{x})}{J_w(\bar{x})} \forall \bar{x} \neq 0
+.. math:: c_{\mathrm{Co^{2+}},p}(\bar{x}) = \frac{j_{\mathrm{Co^{2+}}}(\bar{x})}{J_w(\bar{x})} \forall \bar{x} \neq 0
+
+Relate the concentrations at each phase interface:
+
+.. math:: c_{\mathrm{Li^+},r}(\bar{x})=c_{\mathrm{Li^+},m}(\bar{x},\bar{z}=0)
+.. math:: c_{\mathrm{Co^{2+}},r}(\bar{x})=c_{\mathrm{Co^{2+}},m}(\bar{x},\bar{z}=0)
+.. math:: c_{\mathrm{Cl^-},r}(\bar{x})=c_{\mathrm{Cl^-},m}(\bar{x},\bar{z}=0)
+.. math:: c_{\mathrm{Li^+},m}(\bar{x},\bar{z}=1)=c_{\mathrm{Li^+},p}(\bar{x})
+.. math:: c_{\mathrm{Co^{2+}},m}(\bar{x},\bar{z}=1)=c_{\mathrm{Co^{2+}},p}(\bar{x})
+.. math:: c_{\mathrm{Cl^-},m}(\bar{x},\bar{z}=1)=c_{\mathrm{Cl^-},p}(\bar{x})
+
+Initial and boundary conditions:
+
+.. math:: q_r(\bar{x}=0)=q_f+q_d
+.. math:: q_r(\bar{x}=0) c_{\mathrm{Li^+},r}(\bar{x}=0)=q_f c_{\mathrm{Li^+},f} + q_d c_{\mathrm{Li^+},d}
+.. math:: q_r(\bar{x}=0) c_{\mathrm{Co^{2+}},r}(\bar{x}=0)=q_f c_{\mathrm{Co^{2+}},f} + q_d c_{\mathrm{Co^{2+}},d}
+.. math:: q_p(\bar{x}=0) = 0
+.. math:: c_{\mathrm{Li^+},p} (\bar{x}=0) = 0
+.. math:: c_{\mathrm{Co^{2+}},p} (\bar{x}=0) = 0
+.. math:: \frac{\mathrm{d}q_r(\bar{x})}{\mathrm{d}\bar{x}}(\bar{x}=0)=0
+.. math:: \frac{\mathrm{d}c_{\mathrm{Li^+},r}(\bar{x})}{\mathrm{d}\bar{x}}(\bar{x}=0)=0
+.. math:: \frac{\mathrm{d}c_{\mathrm{Co^{2+}},r}(\bar{x})}{\mathrm{d}\bar{x}}(\bar{x}=0)=0
+
```

Comment:
@lbianchi-lbl , is it possible to move this documentation to a `.rst` file? I'm personally not a fan of extensive documentation inside a file or function docstring, but I don't know how PrOMMiS's documentation is set up, nor which Diataxis category to put this in.

(This is minor feedback, and my mind is open to being changed about documentation organization.)